src/io/filesystem/zip_filesystem.cc

```diff
@@ -518,7 +518,7 @@ StreamWrite* ZipFilesystem::open_stream_write(const std::string& fname) {
 	return new ZipStreamWrite(zip_file_);
 }
 
-void ZipFilesystem::fs_rename(const std::string&, const std::string&) {
+void ZipFilesystem::fs_rename(const std::string& /*unused*/, const std::string& /*unused*/) {
```

Comment:
seems some comments from yesterday got lost, sorry. Can we name this like in ai.